lib/config/config_test.go

```diff
@@ -173,33 +174,28 @@ func TestDeviceConfig(t *testing.T) {
 
 		cfg := wr.Wrapper.(*wrapper).cfg
 
-		expectedFolders := []FolderConfiguration{
-			{
-				ID:               "test",
-				FilesystemType:   FilesystemTypeBasic,
-				Path:             "testdata",
-				Devices:          []FolderDeviceConfiguration{{DeviceID: device1}, {DeviceID: device4}},
-				Type:             FolderTypeSendOnly,
-				RescanIntervalS:  600,
-				FSWatcherEnabled: false,
-				FSWatcherDelayS:  10,
-				Copiers:          0,
-				Hashers:          0,
-				AutoNormalize:    true,
-				MinDiskFree:      Size{1, "%"},
-				MaxConflicts:     -1,
-				Versioning: VersioningConfiguration{
-					Params: map[string]string{},
-				},
-				WeakHashThresholdPct: 25,
-				MarkerName:           DefaultMarkerName,
-				JunctionsAsDirs:      true,
-				MaxConcurrentWrites:  maxConcurrentWritesDefault,
-				XattrFilter: XattrFilter{
-					Entries: []XattrFilterEntry{},
-				},
-			},
-		}
+		fc := FolderConfiguration{}
+		structutil.SetDefaults(&fc)
+		fc.ID = "test"
+		fc.FilesystemType = FilesystemTypeBasic
+		fc.Path = "testdata"
+		fc.Devices = []FolderDeviceConfiguration{{DeviceID: device1}, {DeviceID: device4}}
+		fc.Type = FolderTypeSendOnly
+		fc.RescanIntervalS = 600
+		fc.FSWatcherEnabled = false
+		fc.FSWatcherDelayS = 10
+		fc.Copiers = 0
+		fc.Hashers = 0
+		fc.AutoNormalize = true
+		fc.MinDiskFree = Size{1, "%"}
+		fc.MaxConflicts = -1
+		fc.Versioning.CleanupIntervalS = 1
```

Comment:
Is this on purpose? There is also `fc.Versioning.CleanupIntervalS = 3600` a few lines below.

Reply:
Right, should just set CleanupIntervalS once!